bundle/compliance/policy/data_adapters/file.rego

```diff
@@ -0,0 +1,32 @@
+package compliance.policy.data_adapters.file
```

Comment:
I think it could be nicer to have a `data_adapter.rego` under the `file` directory.
This way we know we didn't forget to implement the relevant data adapter for example.
Also this will let you have one hierarchy less which is more convenient when importing (remove `audit`).